app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: str) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+    for one in people:
+        Person(one["name"], one["age"])
+
+    for one in people:
+        if "wife" in one and one["wife"] is not None:
+            Person.people[one["name"]].wife = Person.people[one["wife"]]
```

Comment:
You could simplify the if condition by using `get`:
```suggestion
        if one.get("wife"):
            Person.people[one["name"]].wife = Person.people[one["wife"]]
```
Besides that, you could use walrus operator to retrieve, evaluate and save the name:
```suggestion
        if wife_name := one.get("wife"):
            Person.people[one["name"]].wife = Person.people[wife_name]
```